includes/polling/applications/ups-nut.inc.php

```diff
@@ -20,36 +20,53 @@
 *
 */
 
+// (2016-11-25, R.Morris) ups-nut, try "extend" -> if not, fall back to "exec" support.
+// -> Similar to approach used by Distro, but skip "legacy UCD-MIB shell support"
+//
 //NET-SNMP-EXTEND-MIB::nsExtendOutputFull."ups-nut"
 $name = 'ups-nut';
 $app_id = $app['app_id'];
 $oid = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.7.117.112.115.45.110.117.116';
 $ups_nut = snmp_get($device, $oid, '-Oqv');
 
+// If "extend" (used above) fails, try "exec" support.
+// Note, exec always splits outputs on newline, so need to use snmp_walk (not a single SNMP entry!)
+if (!$ups_nut) {
+    // Data is in an array, due to how "exec" works with ups-nut.sh output, so snmp_walk to retrieve it
+    $oid = '.1.3.6.1.4.1.2021.7890.2.101';
+    $ups_nut=snmp_walk($device, $oid, '-Oqv');
+}
+//print_r(array_values(explode("\n", $ups_nut)));
```

Comment:
Can we please get rid of the commented debug lines, or replace them with debug versions?  There are a few more below as well.